src/chrome/chromeUtils.ts

```diff
@@ -170,7 +172,7 @@ export function remoteObjectToValue(object: CDTP.Runtime.RemoteObject, stringify
             }
         } else {
             if (object.description === undefined) {
-                throw new Error(`Expected an object that is neither objecr, not function nor undefined to have a description yet it didn't: ${JSON.stringify(object)}`);
+                throw new Error(`Expected an object that is neither object, not function nor undefined to have a description yet it didn't: ${JSON.stringify(object)}`);
             }
 
             // The value is a primitive value, or something that has a description (not object, primitive, or undefined). And force to be string
```

Comment:
I thought it was supposed to be handled by this block

Reply:
Are you sure the exception is correct? I guess that isn't in master

Reply:
I missed that before. Changing implementation